src/options.coffee

```diff
@@ -131,4 +131,32 @@ options =
   version:
     description: "Show version number.\n"
 
+  'hook-worker-timeout':
```

Comment:
This is amazing. Would you mind add the description to all of them and updating the documentation of the CLI and JS interface.